### PR TITLE
[Sharktank] Remove use_attention_mask

### PR DESCRIPTION
**Pull request by Alex-Vasile**
Unconditionally create the attention mask. Creation will be moved inside the attention layer in future PR.

### CODE REVIEW

**Review by rsuderman — APPROVED** [review]

**Comment by Alex-Vasile** [comment]
Failure is unrelated.
